problem.go

```diff
@@ -250,7 +250,7 @@ func readLine() ([]byte, error) {
 	}
 
 	if prefix {
-		return nil, fmt.Errorf("Too long input")
+		return nil, fmt.Errorf("too long input")
```

Comment:
> Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings